src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -1,61 +1,47 @@
 package edu.ucar.unidata.rosetta.controller;
 
+import edu.ucar.unidata.rosetta.domain.Data;
+import edu.ucar.unidata.rosetta.service.DataManager;
+import edu.ucar.unidata.rosetta.service.ResourceManager;
+import edu.ucar.unidata.rosetta.service.validators.CFTypeValidator;
+import edu.ucar.unidata.rosetta.service.validators.FileValidator;
+
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Enumeration;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Enumeration'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)